library/ZendService/Twitter/Twitter.php

```diff
@@ -318,7 +336,8 @@ public function accountVerifyCredentials()
     {
         $this->init();
         $response = $this->get('account/verify_credentials');
-        return new Response($response);
+        $this->twitterResponse->populate($response);
```

Comment:
This goes hand in hand with the feedback above: please don't change the value at runtime, and just create a new `Response` object by just calling the constructor instead. Don't overwrite existing values by changing that internal value, as that just propagates shared mutable state, increasing bug likeliness.